### ISSUE TITLE
[Bug]: Stops are not working

### ISSUE THREAD

**Issue by miku448** [PygmalionAI/aphrodite-engine#742]
### Your current environment

<details>
<summary>The output of `python env.py`</summary>
```text
Your output of `python env.py` here
```

</details>

### 🐛 Describe the bug

It looks like the `stop` parameter is being ignored. I'm using a `openai.api_server` endpoint

my python code:
```
client = OpenAI(
    api_key=openai_api_key,
    base_url=openai_api_base,
)

completion = client.completions.create(
    model="meta-llama/llama-3-8b-instruct",
    prompt="Once upon a time",
    temperature=1.1,
    stop=["was"],
    extra_body={"min_p": 0.1}
)

print("Completion result:", completion)
```
However the result continues to stream tokens after `"was"`
```
Completion result: Completion(id='cmpl-306530abc8f346c3a9acf0321376379e', choices=[CompletionChoice(finish_reason='length', index=0, logprobs=None, text=', in the 15th century, there was a powerful kingdom called Kilk', stop_reason=None)], created=1726590922, model='meta-llama/llama-3-8b-instruct', object='text_completion', system_fingerprint=None, usage=CompletionUsage(completion_tokens=16, prompt_tokens=4, total_tokens=20, completion_tokens_details=None))
```

However the sampling param seems to be arriving well in the aphrodite server:
```
INFO:     Received request cmpl-306530abc8f346c3a9acf0321376379e-0: prompt: ,
params: SamplingParams(temperature=1.1, min_p=0.1, stop=['was']),
prompt_token_ids: [], lora_request: None, prompt_adapter_request: None.
```

**Comment by AlpinDale** [comment]
This has been fixed as of #727, when did you last update?

**Comment by miku448** [comment]
oh sorry, I didn't catch that. I had the last update the day before. thanks!
